### ISSUE TITLE
NodeJS Order Stream Linked Orders bug?

### ISSUE THREAD

**Issue by saulfautley** [BitMEX/api-connectors#124]
I'm getting some strange behaviour in the order stream with the node.js bitmex-realtime-api module when using linked orders.

The issue seems to appears specifically when a limit order fills and closes a position, automatically cancelling a stop market order that is linked via `OneCancelsTheOther`. I believe it has also happened the other way around, i.e. if the stop order is triggered cancelling the limit order.

Basically the order stream data seems to get confused when this happens. For example it will show that there is an open limit order or stop order, even though there are no open orders in the account, and it will never update to correct itself. It's like it misses an update action somewhere.

I've tried for hours to debug this issue, but my node.js and debugging skills are not quite there yet...

**Comment by ryanfox** [comment]
When you say "the order stream" - do you mean the websocket client? If you look in the web UI, does it show the paired order is canceled?

One possible gotcha is that `OneCancelsTheOther` needs to be set on both orders, as well as the same `clOrdLinkID`.

**Comment by saulfautley** [comment]
The linked orders work fine. The problem is the web UI shows the paired order as cancelled, but the websocket client does not.

**Comment by ryanfox** [comment]
I'm having trouble reproducing this - do you have a way to consistently trigger the issue?
